woodstock/voting/models.py

```diff
@@ -104,16 +104,20 @@ def save(self, *args, **kwargs):
             self.slug = uuslug(self.full_name, instance=self)
         super(MozillianProfile, self).save(*args, **kwargs)
 
-    def get_next_entry(self):
-        qs = MozillianProfile.objects.all()
+    def get_next_entry(self, qs=None):
```

Comment:
In order for this PR to work you need to update the `__str__ method of this model to return the `slug` and not the `full_name`.